XenAdmin/MainWindow.cs

```diff
@@ -2664,29 +2671,18 @@ private void Updates_CollectionChanged(CollectionChangeEventArgs e)
                 });
         }
 
-        private void UpdatesCheck_Completed(bool suceeded, string err)
+        private void UpdatesCheck_Completed(bool succeeded, string err)
         {
-
             Program.Invoke(this, () =>
             {
                 updateAlert = Updates.UpdateAlerts.FirstOrDefault(update => update is ClientUpdateAlert) as ClientUpdateAlert;
-                DisableAndHideUpdateClientToolStripMenuItem(updateAlert != null);
+                updateClientToolStripMenuItem.Visible = updateAlert != null;
             });            
         }
 
-        /// <summary>
-        /// Changes disabled state and visibility of the update client button
-        /// </summary>
-        /// <param name="enable">Whether to hide and disable the update client button or not</param>
-        private void DisableAndHideUpdateClientToolStripMenuItem(bool enable)

```

Comment:
I remove this because changed of enabled is not needed if we change the visibility.